template.yaml

```diff
@@ -40,6 +40,14 @@ Globals:
     OpenApiVersion: 3.0.1
 
 Resources:
+
+  SqsResources:
+    Type: AWS::Serverless::Application
+    Properties:
+      Location: ./infrastructure/sqs.yaml
```

Comment:
As discussed, need to de-nest, now we've found what happens in higher environments